src/strange_poker.c

```diff
@@ -41,10 +44,33 @@ int main(int argc, char *argv[])
                 ui->messageUser( ui->data, &ctx, "No players to play!" );
                 continue;
             }
+            
+            //dealing cards
+            
+            for(int i=0; i<52;i++){
+                cards[i]='0';
+            }
+            for (uint32_t i = 0; i < playerCnt; i++){
+                for (int j=0; j<5;j++){
+                    int c=rand() % 52;
+                       while(cards[c]!='0') c=rand() % 52;
+                        players[i].cards[j].suit=c/4;
+                        players[i].cards[j].value=(c%13)+2;
+                        cards[c]='1';
```

Comment:
there are 4 suits and 13 values. You generate 13 suits and 4 values.
If you want i have created pull request onto your fork so you could just merge it you your main if you want.